modal_proto/api.proto

```diff
@@ -621,6 +621,7 @@ message DataChunk {
   oneof data_oneof {
     bytes data = 2;
     string data_blob_id = 3;
+    bool is_eof = 5; // Used for sandbox stdin
```

Comment:
one approach we considered is to use a hardcoded `UUID` to represent an `is_eof` in the context of Sandbox. @ekzhang do you have a preference?